package.json

```diff
@@ -90,21 +90,21 @@
     "nav-frontend-spinner-style": "1.0.2",
     "nav-frontend-typografi": "3.0.4",
     "nav-frontend-typografi-style": "1.0.35",
-    "openid-client": "4.2.2",
+    "openid-client": "5.1.2",
```

Comment:
Denne har jeg fått et problem med å oppradere i de andre appene(får feil i backend med noe parametre), hvordan gikk det for det?

Reply:
Det er noe som er ødelagt her, så det funker ikke å deploye en gang 🤷‍♂️